### ISSUE TITLE
Upgraded Ubuntu 20.04 to 22.04. Terminator icon isnt updated in ubuntu dock.

### ISSUE THREAD

**Issue by usman1515** [gnome-terminator/terminator#597]
**Describe the bug**
The terminator dock icon doesn't seem to match the menu icon in Ubuntu 22.04 LTS when it's opened. This didn't happen in Ubuntu 20.04 LTS.

**To Reproduce**
Simply open terminator in Ubuntu 22.04 LTS.

**Expected behavior**

**Screenshots**
dock icon: 
![icon 1](https://user-images.githubusercontent.com/51785664/166579388-2642a3f5-0758-4c7c-92a4-f38f7e15bc3e.png)

menu icon:
![icon 2](https://user-images.githubusercontent.com/51785664/166579623-61ae7981-5bae-480c-88b4-961ae1160c05.png)

**Desktop:**
 - OS or Linux Distribution: Ubuntu
 - Version: 22.04 LTS
 - Display Technology: Wayland

**Comment by mattrose** [comment]
Not sure what happened here.  It looks like the upgrade may have broken it?  have you tried reinstalling?  I know it doesn't happen on a fresh install of 22.04.

**Comment by enoch85** [comment]
Sorry, I was a bit too early... It's still there:

![image](https://user-images.githubusercontent.com/4511254/169385616-9ee82da6-5497-40b3-83c1-e16c3348a2ad.png)

**Comment by usman1515** [comment]
> Sorry, I was a bit too early... It's still there:
> 
> ![image](https://user-images.githubusercontent.com/4511254/169385616-9ee82da6-5497-40b3-83c1-e16c3348a2ad.png)

Yes, I also double-checked but the issue still persists.

**Comment by mattrose** [comment]
I'm not sure how the new 22.04 dock works these days.  It may be getting confused because terminator doesn't spawn a new process when it opens a new window.  Have you opened a bug with ubuntu?  They may have some ideas on how to fix this.  My work desktop is ubuntu 21.10 and I plan to upgrade to it when I have time, so hopefully I'll be able to track down this behaviour better in the future.

**Comment by mattrose** [comment]
I finally got an ubuntu 22.04 desktop and am unable to reproduce this 
![Screenshot from 2022-06-26 21-26-05](https://user-images.githubusercontent.com/65013/175844079-0a1428e2-923c-48b6-8f4e-6789f2052734.png)

**Comment by mattrose** [comment]
After some consideration, I'm going to close this bug now, as I can't figure out what to do next.  Please re-open if you can find a way for me to reproduce this

**Comment by enoch85** [comment]
@mattrose Please use in day-to-day. Eventually it will error out.

**Comment by mattrose** [comment]
Well, I plan on upgrading from 21.10 to 22.04 on my work desktop so once that happens I can see if that happens, but I need my paid workload to go down a bit before I can devote the time to upgrade.

**Comment by drov0** [comment]
Hello ! I have the same issue, some extra info for @mattrose : 

If I open the terminal by using the shortcut (ctrl + alt + T), it opens with the bad icon:
![image](https://user-images.githubusercontent.com/22356916/185958170-710e122f-85ba-4658-9cfe-a6e778507524.png)

And then subsequent windows all get stored under this bad icon regardless of if I launch them using the shortcut or by clicking the icon in the launcher.

BUT

if I open terminator from the launcher first, I get the correct icon, and then all subsequent terminal launches (done with the shortcut or from the launcher) are under the correct icon.

**Comment by yurenkov** [comment]
Hi everybody. 
I have solved this problem. Just install a new custom keyboard shortcut. ctrl+alt+t and the "terminator"  command.

**Comment by athalhammer** [comment]
I confirm the workaround by @yurenkov worked also for me (I used ctrl+alt+t a lot).

The issue seems to relate to the `x-terminal-emulator` abstraction. When I ran `sudo update-alternatives --config x-terminal-emulator` and chose `/usr/bin/gnome-terminal.wrapper` that icon didn't get replaced by the gear - also using ctrl+alt+t. Running on 22.10 here and my config [here](https://gist.github.com/athalhammer/63f92673704df3d93acd552d4886ba91)
